test/basic-querying.test.js

```diff
@@ -553,6 +554,132 @@ describe('basic-querying', function() {
       });
     });
 
+    it('supports basic a basic query with GeoPoint "near"', function(done) {
+      User.find({
+        where: {
+          addressLoc: {
+            near: '29.9,-90.07',
+          },
+        },
+      }, function(err, users) {
+        if (err) return done(err);
```

Comment:
Still need to return to stop the execution of this function. Otherwise it'll use un-necessary resources.